src/examples/libpmemobj/pi.c

```diff
@@ -90,15 +98,21 @@ pi_task_construct(PMEMobjpool *pop, void *ptr, void *arg)
 /*
  * calc_pi -- worker for pi calculation
  */
+#ifndef _WIN32
 void *
 calc_pi(void *arg)
+#else
+DWORD WINAPI
```

Comment:
The MT-related changes make this example unreadable IMHO. We should create two main files - one for windows and one for linux, and one actual work related file that does not contain ifdefs.